topology-manager/src/main/java/org/apache/aries/rsa/topologymanager/importer/TopologyManagerImport.java

```diff
@@ -147,7 +150,12 @@ private void importServices(String filter) {
 
     private boolean alreadyImported(EndpointDescription endpoint, Set<ImportRegistration> importRegistrations) {
         for (ImportRegistration ir : importRegistrations) {
```

Comment:
Can't we use java streams?

Reply:
I believe the RSA codebase currently is at Java 8 compatibility, so using streams is possible. In the context of this PR I opted to maintain as. much of the code structure.

Reply:
+1 Switching to streams should be a separate change.

Reply:
I believe there is already a separate PR opened by @amichair (#38)